app/controllers/requests_controller.rb

```diff
@@ -1,7 +1,11 @@
 class RequestsController < ApplicationController
 
   def index
-    @requests = current_user.requests
+    if user_signed_in? 
```

Comment:
You can write this if sentence in one line: 
`user_signed_in? ? @requests = current_user.requests : redirect_to new_user_session_path`